### PR TITLE
Testing

### LINKED ISSUE #74: Scripts timeout
Bonjour,
Je viens d'installer ce package, via la GUI de YNH mais malheureusement, rien ne s'affiche lors du premier lancement ->HTTP ERROR 500
Nextcloud, Transmission, WordPress fonctionnent parfaitement dessus.
Serveur ARM, chez SoYouStart.
Il y a quelques chose a faire, après l'installation ?

Vous avez besoin de quoi comme logs pour debuger
Merci d'avance

### LINKED ISSUE #90: Question : Title, an option ?
Why title is optional ? Because if the package is stable, we could set is as needed.

### PR DESCRIPTION
**Pull request by tituspijean**
This PR adds:
- `php-fpm.ini` to increase time and memory limits that have been encountered by some users
- `change_url` script
- requirement for all installation fields, as automatic post-installation is quite stable.

`README` has been updated accordingly. Build and "Install with YNH" buttons have been added also.

### Will close these issues : 
fixes #74 
fixes #90

### CODE REVIEW

**Comment by tituspijean** [comment]
My review :
- `change_url` tested ✔️  
- new required fields tested ✔️
- `php-fpm.ini` improvements NOT tested ❔, as I do not have a testing environment that triggers the timeouts. At least it does not affect successful installations.

File: README.md [hunk 1]
```diff
@@ -21,19 +26,6 @@ You can also install it with `yunohost app install https://github.com/YunoHost-A
   - `path`
   - `admin`, among the YunoHosts users
   - `public`, *true* by default, for guests to read the forum
-- Optional parameters are :
-  - `title` :warning: if you leave it empty, you will have to perform manual post-installation.
+  - `title` of the forum
```
Comment (frju365) [inline README.md]: as it's required now, not optional ;)
(it's my fault forgot to update README)

File: conf/php-fpm.ini [hunk 2]
```diff
@@ -0,0 +1,10 @@
+; Common values to change to increase file upload limit
+; upload_max_filesize = 50M
```
Comment (frju365) [inline conf/php-fpm.ini:2]: Wonder is one day, we will have to set upload_max_filesize a bit higher than default one.

**Review by frju365 — CHANGES_REQUESTED** [review]
LGTM, Code_Review ok

**Review by frju365 — APPROVED** [review]

**Comment by frju365** [comment]
Don't know what I did with review, but everything seems to be ok. We can merge when you want.

**Comment by frju365** [comment]
I didn't test script timeout too as my server and my laptop are have a high amount of RAM and have no problem with that.

File: conf/php-fpm.ini (hunk 2 above)
Comment (tituspijean) [reply conf/php-fpm.ini:2]: Do you think a forum need that much upload capacity?

**Comment by tituspijean** [comment]
Maybe we can add a note in the `README` to advise users to create a swap if they have little RAM?

File: conf/php-fpm.ini (hunk 2 above)
Comment (frju365) [reply conf/php-fpm.ini:2]: Mh... I don't know what is the default upload size but it's depends on the forum. Sometime user could upload Archive (tar, zip) which could have more than 50MO.

Comment (frju365) [reply conf/php-fpm.ini:2]: It depends on the capabilities of the server too (enough space...).

**Comment by frju365** [comment]
yes, good idea. Like in the package mastodon_ynh : https://github.com/YunoHost-Apps/mastodon_ynh/blob/master/README.md

**Pull request merged**